src/table/TableWrapper.jsx

```diff
@@ -73,10 +75,44 @@ export default function TableWrapper(props) {
     };
   }, []);
 
+  useEffect(() => {
+    if (focusedCellCoordsState[0] < 2) {
+      tableSection.current.scrollTo({
+        top: 0,
+        behavior: 'smooth',
+      });
+    } else if (isMovingTop) {
+      setIsMovingTop(false);
+
+      const rowElements = rootElement.getElementsByClassName('sn-table-row');
+      const [x, y] = focusedCellCoordsState;
+      const cell = rowElements[x]?.getElementsByClassName('sn-table-cell')[y];
+
+      if (cell.offsetTop - cell.offsetHeight * 2 < tableSection.current.scrollTop) {
+        tableSection.current.scrollTo({
+          top: tableSection.current.scrollTop - cell.offsetHeight || 0,
+          behavior: 'smooth',
+        });
+      }
+    }
+  }, [tableSection, focusedCellCoordsState, isMovingTop, rootElement]);
```

Comment:
Do we want to listen to all of these? e.g. pretty sure `rootElement` wont change unless you remount the whole thing. Why isn't just `focusedCellCoordsState` enough?

Reply:
Fixed ✅
I think probably i was afraid of some eslint related error ( by habit ) that i put it there